tests/loss/test_ibot_loss.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import torch
+
+from lightly.loss.ibot_loss import IBOTPatchLoss
+
+
+class TestIBOTPatchLoss:
+    @pytest.mark.parametrize("device", ["cpu", "cuda"])
+    def test_forward(self, device: str) -> None:
+        if not torch.cuda.is_available() and device == "cuda":
+            pytest.skip("CUDA not available")
+
+        criterion = IBOTPatchLoss(output_dim=2, teacher_temp=0.1, student_temp=0.2)
+        teacher_out = torch.tensor([[0.1, 0.2], [0.3, 0.4], [0.5, 0.6]])
+        student_out = torch.tensor([[0.7, 0.8], [0.9, 1.0], [1.1, 1.2]])
+        mask = torch.tensor(
+            [[[True, False], [True, False]], [[False, False], [False, True]]]
+        )
```

Comment:
I am missing a test where the clamping is actually needed as there are 0 times True for one image.

Reply:
Updated, had to change the expected loss as adding more entries to the mask increases the batch size.